evap/contributor/tests/test_views.py

```diff
@@ -25,7 +25,7 @@ def test_direct_delegation_request(self):
 
         self.assertContains(
             page,
-            "User {} was added as a contributor for course {} and was sent an email with further information.".format(str(self.non_responsible), str(self.course))
+            "{} was added as a contributor for course {} and was sent an email with further information.".format(str(self.non_responsible), str(self.course))
```

Comment:
got distracted again? ;) you didn't put the `"`s here, now the tests are failing.